tldw_Server_API/app/api/v1/endpoints/data_tables.py

```
@@ -0,0 +1,1126 @@
+    )
+    total = db.count_data_tables(
+        status=status_filter,
+        search=search,
+        owner_user_id=owner_user_id,
+    )
+    table_ids = []
+    for row in rows:
+        try:
+            table_ids.append(int(row.get("id")))
+        except Exception:
+            continue
```

Ruff (0.14.13)

`try`-`except`-`continue` detected, consider logging the exception

(S112)